packages/super-components/components/msc-billing-tile/src/msc-billing-tile/msc-billing-renewal.tsx

```diff
@@ -138,18 +135,15 @@ export class MscBillingRenewal implements IMSCBillingRenewal {
             <osds-link
               data-tracking={this.manageRenewDataTracking}
               target="_blank"
```

Comment:
Should change it to target={OdsHTMLAnchorElementTarget._blank}

Reply:
done thanks